nsqadmin/static/js/views/app.js

```diff
@@ -136,6 +136,10 @@ var AppView = BaseView.extend({
     },
 
     onLinkClick: function(e) {
+        if (e.ctrlKey || e.metaKey) {
+          // allow ctrl+click to open in a new tab
+          return true
```

Comment:
I don't know if you actually need to return `true` here. 

Also, missing semicolon usage and indentation isn't consistent with rest of file.